test/runtests.jl

```diff
@@ -3,6 +3,11 @@ using MakieGallery
 using Test
 using GLMakie
 using StaticArrays, GeometryBasics
+using Observables
```

Comment:
We should reexport all these utility packages (Colors, Observables)

Reply:
yes, `onany` and `Gray` are two missing exports that always annoy me :)